<?xml version="1.0" encoding="UTF-8"?><testrun name="IntegrationTest" project="bank-example" tests="5" started="5" failures="0" errors="2" ignored="0">
  <testsuite name="com.aprilsulu.bank.IntegrationTest" time="0.383">
    <testcase name="testGetIndividualAccount" classname="com.aprilsulu.bank.IntegrationTest" time="0.343"/>
    <testcase name="testGetAccounts" classname="com.aprilsulu.bank.IntegrationTest" time="0.022"/>
    <testcase name="testGetIndividualAccountNotFound" classname="com.aprilsulu.bank.IntegrationTest" time="0.012"/>
    <testcase name="testTransferIdNotFound" classname="com.aprilsulu.bank.IntegrationTest" time="0.005">
      <error>javax.ws.rs.ProcessingException: java.net.MalformedURLException: For input string: "64033accounts"&#13;
	at org.glassfish.jersey.client.internal.HttpUrlConnector.apply(HttpUrlConnector.java:287)&#13;
	at org.glassfish.jersey.client.ClientRuntime.invoke(ClientRuntime.java:252)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$1.call(JerseyInvocation.java:684)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$1.call(JerseyInvocation.java:681)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:228)&#13;
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:444)&#13;
	at org.glassfish.jersey.client.JerseyInvocation.invoke(JerseyInvocation.java:681)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$Builder.method(JerseyInvocation.java:437)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$Builder.post(JerseyInvocation.java:343)&#13;
	at com.aprilsulu.bank.IntegrationTest.testTransferIdNotFound(IntegrationTest.java:92)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)&#13;
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)&#13;
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.net.MalformedURLException: For input string: "64033accounts"&#13;
	at java.net.URL.&lt;init&gt;(Unknown Source)&#13;
	at java.net.URL.&lt;init&gt;(Unknown Source)&#13;
	at java.net.URL.&lt;init&gt;(Unknown Source)&#13;
	at java.net.URI.toURL(Unknown Source)&#13;
	at org.glassfish.jersey.client.internal.HttpUrlConnector._apply(HttpUrlConnector.java:340)&#13;
	at org.glassfish.jersey.client.internal.HttpUrlConnector.apply(HttpUrlConnector.java:285)&#13;
	... 37 more&#13;
Caused by: java.lang.NumberFormatException: For input string: "64033accounts"&#13;
	at java.lang.NumberFormatException.forInputString(Unknown Source)&#13;
	at java.lang.Integer.parseInt(Unknown Source)&#13;
	at java.lang.Integer.parseInt(Unknown Source)&#13;
	at java.net.URLStreamHandler.parseURL(Unknown Source)&#13;
	... 43 more&#13;
&#13;
</error>
    </testcase>
    <testcase name="testTransfer" classname="com.aprilsulu.bank.IntegrationTest" time="0.001">
      <error>javax.ws.rs.ProcessingException: java.net.MalformedURLException: For input string: "64033accounts"&#13;
	at org.glassfish.jersey.client.internal.HttpUrlConnector.apply(HttpUrlConnector.java:287)&#13;
	at org.glassfish.jersey.client.ClientRuntime.invoke(ClientRuntime.java:252)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$1.call(JerseyInvocation.java:684)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$1.call(JerseyInvocation.java:681)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:228)&#13;
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:444)&#13;
	at org.glassfish.jersey.client.JerseyInvocation.invoke(JerseyInvocation.java:681)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$Builder.method(JerseyInvocation.java:437)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$Builder.post(JerseyInvocation.java:343)&#13;
	at com.aprilsulu.bank.IntegrationTest.testTransfer(IntegrationTest.java:72)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)&#13;
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)&#13;
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.net.MalformedURLException: For input string: "64033accounts"&#13;
	at java.net.URL.&lt;init&gt;(Unknown Source)&#13;
	at java.net.URL.&lt;init&gt;(Unknown Source)&#13;
	at java.net.URL.&lt;init&gt;(Unknown Source)&#13;
	at java.net.URI.toURL(Unknown Source)&#13;
	at org.glassfish.jersey.client.internal.HttpUrlConnector._apply(HttpUrlConnector.java:340)&#13;
	at org.glassfish.jersey.client.internal.HttpUrlConnector.apply(HttpUrlConnector.java:285)&#13;
	... 37 more&#13;
Caused by: java.lang.NumberFormatException: For input string: "64033accounts"&#13;
	at java.lang.NumberFormatException.forInputString(Unknown Source)&#13;
	at java.lang.Integer.parseInt(Unknown Source)&#13;
	at java.lang.Integer.parseInt(Unknown Source)&#13;
	at java.net.URLStreamHandler.parseURL(Unknown Source)&#13;
	... 43 more&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
